aces/Loris
# [SQL] Removed mri_acquisition_dates table from the database and the LORIS code

## Brief summary of changes

This PR removes the `mri_acquisition_dates` table from the database as this table does not necessarily store the correct acquisition date. Instead, replaced all the queries to look for the minimal date present in the `parameter_file` table which is more robust than what was stored in `mri_acquisition_dates`.

Note: the decision to remove this table was taken during an imaging meeting in 2017 and confirmed at an other imaging meeting in 2019. 

Link to the MRI PR: https://github.com/aces/Loris-MRI/pull/463

#### Testing instructions (if applicable)

1. Run the SQL patch `SQL/New_patches/2019-07-17_remove_mri_acquisition_dates_table.sql`
2. Make sure the dashboard is loading and plotting the correct graph for the MRI acquisitions
3. Make sure the imaging browser is loading and that the First Acquisition date properly populated

#### Links to related tickets (GitHub, Redmine, ...)

* https://redmine.cbrain.mcgill.ca/issues/6040
